sdks/python/apache_beam/io/gcp/gcsio.py

```diff
@@ -87,6 +87,50 @@
 MAX_BATCH_OPERATION_SIZE = 100
 
 
+def ProxyInfoFromEnvironmentVar(proxy_env_var):
```

Comment:
<!--new_thread; commit:a648a0224e9c7cc59c6d65ae14b03e07ee1e2171; resolved:0-->
Please use PEP8 function names thoughout (https://www.python.org/dev/peps/pep-0008/#function-names).

Reply:
Sorry for the camel case :) reformatting with underscores to keep consistency.